### PR TITLE
project: Improve frontend support for applications

### PR DESCRIPTION
**Pull request by hemangsk**

### CODE REVIEW

File: public/javascripts/dashboard.js [hunk 1]
```diff
@@ -119,41 +119,69 @@ app.controller('fc', function ($scope, $http) {
...
+                    if(data.data.length == 1) {
                         $scope.cur_candidate = mapper(data.data[0]);
+                        console.log('length 1')
+                        console.log($scope.cur_candidate)
+                        $scope.setCurCandidates(mapper(data.data[0]));
+
                     } else {
                         $scope.cur_candidate = $scope.candidate;
                     }
+
```
Comment (hemangsk) [inline public/javascripts/dashboard.js:164]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `cli`.*

The issue can be fixed by applying the following patch:

```diff
--- a/public/javascripts/dashboard.js
+++ b/public/javascripts/dashboard.js
@@ -161,7 +161,7 @@
                     } else {
                         $scope.cur_candidate = $scope.candidate;
                     }
-                    
+
 
                 })
         }
```

**Review by surender7790 — APPROVED** [review]
great job 👍

**Comment by surender7790** [comment]
@rultor merge

**Comment by surender7790** [comment]
ack e94b42e

**Pull request merged**